fedn/fedn/common/control/package.py

```diff
@@ -215,7 +215,7 @@ def dispatcher(self, run_path):
         from_path = os.path.join(os.getcwd(), 'client')
         
         from distutils.dir_util import copy_tree
-        copy_tree(from_path, run_path)
+        copy_tree(from_path, run_path, preserve_times=False)
```

Comment:
Maybe add a comment / note about why that flag is set in case we need to come back to it later.